python/tests/toolchains/run_acceptance_test.py.tmpl

```diff
@@ -53,6 +53,15 @@ class TestPythonVersion(unittest.TestCase):
 
         subprocess.run("bazel test //...", shell=True, check=True)
 
+        stream = os.popen("bazel info execution_root")
```

Comment:
Please use the subprocess module instead. Right now, this is leaking an opened file description, which spams the logs with a warning.